protocol/testutil/app/app.go

```diff
@@ -703,7 +710,7 @@ func MustMakeCheckTxWithPrivKeySupplier(
 		rand.New(rand.NewSource(42)),
 		app.TxConfig(),
 		messages,
-		sdk.Coins{},
+		options.FeeAmt,
```

Comment:
nit: why is this needed?

Reply:
Commented. Needed for submitting transaction that require gas fees (e.g. gov proposals)